about.html

```diff
@@ -0,0 +1,192 @@
+            <p id="p_contact" class="text-dark">Please contact us per Email for any further questions</p>
+            <small class="text-dark">vedic2023@sidereal.com</small>
+          </div>
+        </article>
+      </main>
+    </div>    
+  </div>
+
+  <section id="logoSection" class="displayGrid align-center">
+    <h1 class="headline">Regional Vedic Astrology Workshop 2023 Logo</h1>
+    <hr>
+    
+    <div id="advertise" class="text-description">
+      <p>Lorem ipsum dolor sit amet consectetur. Laboriosam quos repellendus quod!</p>
+    </div>
+    <div class="logo-img displayFlex">
+      <img id="img_logo" src="images/logo.png" alt="website logo">
+    </div>
+  </section>
+  
+  <hr id="hr1" class="w-100">
+
+  <section id="past" class="displayGrid align-center">
+    <h1 class="headline">See the past Regional Workshop</h1>
+    <hr>
+    
+    <p class="text-description text-dark">
+      Lorem ipsum dolor sit amet consectetur.
+    </p>
+    <div id="archives">
+      <ul id="archiveList">
+        <li>
+          <a href="#">
+            <img src="images/archive-1.png" alt="archive image 1" class="w-100">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/archive-2.png" alt="archive image 2" class="w-100">
+          </a>
+        </li>
+      </ul>
+    </div>
+  </section>
+
+  <section id="partner" class="displayGrid align-center aboutShowHide">
+    <h1 class="headline text-grey">Partner</h1>
+    <hr>
+    
+    <div id="partners">
+      <ul id="partnerList" class="displayFlex">
+        <li>
+          <img src="images/partner/mozilla-logo.png" alt="naver logo" class="w-100">
+        </li>
+        <li>
+          <img src="images/partner/google-logo.png" alt="google logo" class="w-100">
+        </li>
+        <li>
+          <img src="images/partner/naver-logo.png" alt="naver logo" class="w-100">
+        </li>
+        <li>
+          <img src="images/partner/daumkakaologo-logo.png" alt="naver logo" class="w-100">
+        </li>
+        <li>
+          <img src="images/partner/airbnb-logo.png" alt="airbnb logo" class="w-100">
+        </li>
+      </ul>
+    </div>
+  </section>
+
+  <footer id="aboutFooter">
+    <div class="displayFlex footer">
+      <img src="images/cck-logo.png" alt="Ruby Astro Logo" class="footer-logo aboutShowHide">
+      <img id="whiteLogo" src="images/cck-logo-white.png" alt="Ruby Astro Logo" class="footer-logo">
+      <p id="yearNow">&#169;2023 Ruby Astro <span>Some Rights Reserved</span></p>
+    </div>
+  </footer>
+
+  <!-- Modal Settings Starts -->
+  <section id="modalSetting" class="modalContainer">
+    <div id="modalContentSetting" class="modalContent">
+      <button type="button" id="btnCloseNav">
+        <img src="images/cross2.png" alt="Hide Popup">
+      </button>
+      <div id="div_menu">
+        <ul id="menuList" class="modalNav">
+          <li>
+            <a class="text-white" href="#home">Home</a>
+          </li>
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly I can't switch `About Page` to `Home Page` on mobile version, please fix it 🙏🏼 

![mobile](https://github.com/rubydevi/the-korner-capstone/assets/63915024/3aae67a9-207a-49b3-8c14-3cd224815c36)